create-security-certificates.md

```diff
@@ -2,3 +2,68 @@
 title: Create Security Certificates
 toc: false
 ---
+
+A secure CockroachDB cluster uses [TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security) for encrypted inter-node and client-node communication and requires CA, node, and client certificates and keys. To create these certificates and keys, use the `cockroach cert` command with the appropriate subcommands and flags. 
+
+When using <code>cockroach cert</code> to create node and client certificates, you will need access to a local copy of the CA certificate and key. It is therefore recommended to create all certificates and keys in one place and then distribute node and client certificates and keys appropriately. For the CA key, be sure to store it somewhere safe and keep a backup; if you loose it, you will not be able to add new nodes to your cluster. For a walkthrough of this process, see [Manual Deployment](manual-deployment.html).
```

Comment:
s/"if you loose it"/"if you lose it"/

s/"you will not be able to add new nodes"/"you will not be able to add new nodes or clients"/